components/SignForm/LoginForm.tsx

```diff
@@ -0,0 +1,81 @@
+import { useForm } from 'react-hook-form';
+import Button from '../button/Button';
+import styles from './Form.module.scss';
+import { ErrorMessage } from '@hookform/error-message';
+import classNames from 'classnames';
+import { useAuth } from '@/contexts/AuthProvider';
+import RenderField from './RenderField';
+
+function LoginForm() {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<ILoginForm>({
+    mode: 'onChange',
+    defaultValues: {
+      email: '',
+      password: '',
+    },
+  });
+
+  const { login } = useAuth(false);
+
+  const onSubmit = (data: ILoginForm) => {
+    login(data);
+  };
```

Comment:
개인마다 다르겠지만 전 개인적으로 아래 form에서 onSubmit과 handleSubmit 을 익명함수로 사용하는것을 지양합니다.

```suggestion
  const onSubmit = handleSubmit((data)=> {
    login(data);
  });
```
이렇게 할 수도 있다는것만 알아주세요~